internal/common/foragepayoff.go

```diff
@@ -0,0 +1,40 @@
+package common

```

Comment:
notice the filename here, golang does not allow circular imports, and since we need common stuff here, we can just organise code by filename this way... (some people prefer `forage_payoff.go`, but usually we reserve `_` for build targets or tests.